lib/minitest/reporters/nutrio_reporter.rb

```diff
@@ -0,0 +1,141 @@
+require 'ansi/code'
+
+class NutrioReporter < Minitest::Reporters::BaseReporter
```

Comment:
there seems to be a LOT of boilerplate code in this file.
was it copied from somewhere else?
could we just subclass that source instead?